app/helpers/react_components/mentoring/representations/without_feedback.rb

```diff
@@ -46,7 +46,8 @@ def representations
         end
 
         def representations_with_feedback_count
-          Exercise::Representation::Search.(mentor:, with_feedback: true, sorted: false, paginated: false).count
+          0 # TODO: support counting only tracks of which user is supermentor
+          # Exercise::Representation::Search.(mentor:, with_feedback: true, sorted: false, paginated: false).count
```

Comment:
Right now, this would return an invalid count, as we're only showing tracks for which the user is a supermentor (>= 100 solutions mentored), but `Exercise::Representation::Search` doesn't know about those tracks. I could theoretically pass them here, but that would require some more work (not to run the logic more than once) and I want to get this merged to "fix" the issue.